src/api/mock-interceptor.ts

```diff
@@ -176,11 +176,17 @@ export const interceptRequestsOnMock = () => {
   // Institution Registry
   mock.onGet(new RegExp(`${API_URL}${InstitutionApiPaths.INSTITUTIONS}`)).reply(200, mockInstitutionResponse);
   mock
-    .onGet(new RegExp(`${API_URL}${InstitutionApiPaths.DEPARTMENTS}\\?uri=https://api.cristin.no/v2/institutions/194`))
+    .onGet(
+      new RegExp(
+        `${API_URL}${InstitutionApiPaths.DEPARTMENTS}\\?uri=https%3A%2F%2Fapi.cristin.no%2Fv2%2Finstitutions%2F194&language=en`
```

Comment:
huff 😅  du vil ikke slenge på en regex som sjekker på `language=en` på slutten isteden da?

Reply:
dette er faktisk en regex :p skjønner ikke hvor du vil... suggestion?

Reply:
```suggestion
        `$.*194&language=en$`
```
sånn liksom ☝️  eller synes du det blir for lite eksplisitt?